apps/condo/domains/billing/schema/BillingReceiptsService.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Generated by `createservice billing.BillingReceiptsService --type queries`
+ */
+
+const { Resident, ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
+const { BillingReceipt } = require('@condo/domains/billing/utils/serverSchema')
+const { NOT_FOUND_ERROR } = require('@condo/domains/common/constants/errors')
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/billing/access/GetBillingReceiptsForServiceConsumerService')
+
+
+const GetBillingReceiptsForServiceConsumer = new GQLCustomSchema('GetBillingReceiptsForServiceConsumerService', {
+    types: [
+        {
+            access: true,
+            type: 'input BillingReceiptsForServiceConsumerInput { dv: Int!, sender: JSON! serviceConsumerId: ID! }',
+        },
+        {
+            access: true,
+            type: 'type BillingReceiptsForServiceConsumerOutput { id: String!, dv: String!, recipient: JSON!, period: String!, toPay: String!, printableNumber: String!, toPayDetails: JSON!, services: JSON! }',
+        },
+    ],
+    
+    queries: [
+        {
+            access: access.canGetBillingReceiptsForServiceConsumer,
+            schema: 'executeBillingReceiptsForServiceConsumer (data: BillingReceiptsForServiceConsumerInput!): BillingReceiptsForServiceConsumerOutput',
```

Comment:
why `execute`? `getAllResidentBillingReceipts` should return list of `ResidentBullingReceipt` for resident.

Reply:
already fixed